dev/io.openliberty.jakartaee9.internal_fat/fat/src/io/openliberty/jakartaee9/internal/tests/EE9FeatureCompatibilityTest.java

```diff
@@ -130,6 +130,8 @@ private static Set<String> getCompatibleFeatures(Set<String> allFeatures, boolea
         compatibleFeatures.remove("springBoot-2.0");
         compatibleFeatures.remove("springBoot-3.0"); // springBoot 3.0 only supports EE 10
 
+        compatibleFeatures.remove("mpReactiveMessaging-3.0"); //still in development
+
```

Comment:
Why isn't it compatible yet?

Reply:
A lack of testing at the time when this PR was first created. Even now, I have only repeated one test on that.